### PR TITLE
Places

### PR DESCRIPTION
**Pull request by arindambarman1**

### CODE REVIEW

File: test/fixtures/places.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (arindambarman1) [inline test/fixtures/places.yml]: oi eibur ki hoi rey?

File: test/models/place_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PlaceTest < ActiveSupport::TestCase
```
Comment (arindambarman1) [inline test/models/place_test.rb]: eiburu?

File: lib/function.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require "#{Rails.root}/app/models/place"
+
+class Function
+
+	def self.bangalore
```
Comment (arindambarman1) [inline lib/function.rb]: Eikhini part dynamic koribo lagibo niki? Each city kaarne tu beleg beleg funciton likhibo nuaru nooh? So paramater pass koribo lagibo sage noh?

File: db/migrate/20150914211852_add_user_id_to_users.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddUserIdToUsers < ActiveRecord::Migration
```
Comment (arindambarman1) [inline db/migrate/20150914211852_add_user_id_to_users.rb]: Cousin new users table kaarne migration bonua nai niki toi?

File: lib/function.rb (hunk 3 above)
Comment (chaos--theory) [inline lib/function.rb]: Yepp parameter pass koribo lagibo, working on it.

File: db/migrate/20150914211852_add_user_id_to_users.rb (hunk 4 above)
Comment (chaos--theory) [inline db/migrate/20150914211852_add_user_id_to_users.rb]: Oh, sorry eitu migration delete koribo thaki gol bey. Done!

File: lib/function.rb (hunk 3 above)
Comment (arindambarman1) [inline lib/function.rb]: Mosti.

File: db/migrate/20150914211852_add_user_id_to_users.rb (hunk 4 above)
Comment (arindambarman1) [inline db/migrate/20150914211852_add_user_id_to_users.rb]: Thanks be. Etya mur tat kam koribo.

File: test/models/place_test.rb (hunk 2 above)
Comment (arindambarman1) [inline test/models/place_test.rb]: @chaos--theory oi eibur tests niki?

Comment (chaos--theory) [inline test/models/place_test.rb]: Hoi nki, `rails generate model Place` use korisilu karne ahise sake rey.

Comment (arindambarman1) [inline test/models/place_test.rb]: oh okay okay! eibur moi gusai dim. Rspec add kori dim.

File: db/migrate/20150914211852_add_user_id_to_users.rb (hunk 4 above)
Comment (chaos--theory) [inline db/migrate/20150914211852_add_user_id_to_users.rb]: Try kori sasun.

Comment (arindambarman1) [inline db/migrate/20150914211852_add_user_id_to_users.rb]: okay sai asu roh

Regards,
Arindam Barman
arindambarman-twitter
<http://t.sidekickopen16.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX45wvn10W4Xq_GW8qm9fRW8q5y9d56dTXTf76_vTR02?t=https%3A%2F%2Ftwitter.com%2Farindambarman&si=4930011077279744&pi=fe8cbb67-87bc-469f-caf5-fd99ec2cbedc>
+919591374691

**Pull request merged**